tools/package.json

```diff
@@ -1,23 +0,0 @@
-{
-  "name": "template-schema-validation-tools",
-  "version": "1.0.0",
-  "description": "Template schema validation tools.",
-  "author": "Dan Schulte",
-  "license": "MIT",
-  "scripts": {
-    "clean": "rmdir /S /Q out",
-    "test": "mocha runSchemaValidation.js"
-  },
-  "dependencies": {
-    "chalk": "^1.1.3",
-    "sync-request": "^2.1.0",
-    "tv4": "1.2.7",
-    "vscode-json-languageservice": "3.0.2",
-    "mocha": "^2.3.3",
-    "mocha-each": "^1.1.0"
-  },
-  "devDependencies": {
-    "istanbul": "^0.4.4",
-    "mocha": "^2.5.3"
-  }
-}
```

Comment:
why is this file being removed?

Reply:
@TianoMS I think one of the imports I made did it. Anyway I reverted